conftest.py

```diff
@@ -565,63 +584,93 @@ def make_test_dir(caller_function=None, caller_fspath=None, level=1):
             caller_function = caller_frame.function
 
         dir_path = FileUtils._test_dir_path(caller_function, caller_fspath)
-        # dir_path = os.path.join(test_dir, dir_name)
+        if sub_dir:
```

Comment:
@MattToast I gave myself the ability to add subdirectories in the test directory using FileUtils

Reply:
Love it! One request: can we add the new param to the docstr, just for future reference?

Reply:
added sub_dir & level to docstring